<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>754</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>onlineMotionCorrection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsViewRaw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCorrected">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRaw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Raw image</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCorrected">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Corrected</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderCh1Min">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>560</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8191</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderCh1Max">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>580</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
    <property name="value">
     <number>512</number>
    </property>
    <property name="sliderPosition">
     <number>512</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderCh2Min">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>610</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8191</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderCh2Max">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>630</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
    <property name="value">
     <number>8192</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderCh3Min">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>660</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8191</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderCh3Max">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>680</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
    <property name="value">
     <number>8192</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelCh1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ch1 (Green)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCh2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ch2 (Red)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCh3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Template (Blue)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCh1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCh2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCh3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>680</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAverage">
    <property name="geometry">
     <rect>
      <x>1002</x>
      <y>0</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string>10</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelAverage">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>5</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Average frames:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNew">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCh1Min">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>560</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8191</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCh1Max">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>512</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCh2Min">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>610</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8191</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCh2Max">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>630</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>8192</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCh3Min">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>660</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8191</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCh3Max">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>680</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8192</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>8192</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QWidget" name="widgetNew" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>560</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>630</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrent">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>540</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetCurrent" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>560</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
